docs/source/user_guide/automl.ipynb

```diff
@@ -318,6 +318,35 @@
    "source": [
     "automl.results"
    ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## Ensemble methods: Stacking\n",
```

Comment:
Up to you but I think something like "Adding ensemble methods to AutoML" as a header and then "stacking" as a subheader could make it fit better with the rest of the page.

Reply:
Good idea! That would structure would accommodate adding new ensemble algorithms in the future!